opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -343,7 +344,7 @@ message HistogramValue {
 
       // exemplar_attachments are contextual information about the example value.
       // Keys in this list must be unique.
-      repeated StringKeyValuePair attachments = 3;
+      repeated opentelemetry.proto.common.v1.AttributeKeyValue attachments = 3;
```

Comment:
this changes the data model, do we have an agreement that we want to support labels that are not string? This increases the size of the messages (in-memory) without a real benefit (unless you prove me that there are some important systems that support multiple types for labels).

Reply:
My understanding is that this can be any set of attributes that you want to attach to a measurement that can be later explored in the backend UI. I think non-string values can be useful here, just like they are for e.g. Span attributes. It seems to be the same use case, providing additional information to understand what operation is represented (and exemplar is a representation of an operation, isn't it?).